src/lib/idTokenValidator.ts

```diff
@@ -0,0 +1,35 @@
+import * as jose from 'jose';
+import {InvalidIDTokenException} from './exceptions';
+import OAuthAgentConfiguration from './oauthAgentConfiguration';
+
+/*
+ * Make some sanity checks to ensure that the issuer and audience are configured correctly
+ * The ID token is received over a trusted back channel connection so its signature does not need verifying
+ * https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation
+ */
+export function validateIDtoken(config: OAuthAgentConfiguration, idToken: string) {
+
+    const payload = jose.decodeJwt(idToken)
```

Comment:
Same comment as in the next.js implementation. I think that it would be better to write our own method for decoding the JWT and drop the jose dependency.

Reply:
Again I left this for now, to encourage use of a JWT library by customers.